wiki/CreatingOutfits.md

```diff
@@ -47,7 +47,10 @@ Most attributes are given as a single number, but there are a few "special" attr
   * "Power"
   * "Engines"
   * "Hand to Hand"
+  * "Unique"
+  * "Minerals"
   * "Special"
+  * "Licenses"
```

Comment:
I think these should also have version numbers listed for when they first became available.

Reply:
Just added -- all 4 (including "Special") look to have been created with 0.10.12